### PR TITLE
Create blog post

### PR DESCRIPTION
**Pull request by sabaucristina**

### CODE REVIEW

File: frontend/app/components/new-post.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  actions:{
+    createPost(post){
+      //const params=this.getProperties('title', 'description', 'created_by');
```
Comment (shotgundebugging) [inline frontend/app/components/new-post.js]: You can remove this

File: frontend/app/controllers/create-post.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+  actions:{
+    createPost(){
+      const post_item=this.get('model');
+      var self=this;
```
Comment (shotgundebugging) [inline frontend/app/controllers/create-post.js]: Why is this needed? Just use `this`.

File: spec/features/post_item_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+     page.fill_in "description", with: description
+     page.fill_in "created_by", with: created_by
+     expect { click_on 'button' }.
+       to change { PostItem.count }.from(0).to(1)
+
+    post_item = PostItem.find_by_title(title)
+    expect(post_item.description).to eq(description)
+    expect(post_item.created_by).to eq(created_by)
+
+    some_other_page_path='/post-items/1'
```
Comment (shotgundebugging) [inline spec/features/post_item_spec.rb]: Construct the string dynamically: `"post-items/#{post_item.id}"`

File: frontend/app/controllers/create-post.js (hunk 2 above)
Comment (sabaucristina) [reply frontend/app/controllers/create-post.js]: I put this outside of promise.  I want to obtain the content of the controller not of my promise call
